src/views/order_view.js

```diff
@@ -0,0 +1,29 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+
+import Quote from '../models/quote';
+import QuoteList from '../collections/quote_list';
+
+const OrderView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.listenTo(this.model, 'change', this.render);
+    this.bus = params.bus;
+    console.log(`${this.model.attributes.symbol}${this.model.attributes.buy}orderFinished`);
+    this.listenTo(this.bus, `${this.model.attributes.symbol}${this.model.attributes.buy}orderFinished`, this.cancelOrder);
```

Comment:
By having this two step process of triggering and listening, it's quite possible that an `Order` could execute twice.